clientd3d/color.c

```diff
@@ -403,19 +404,25 @@ HBRUSH DialogCtlColor(HWND hwnd, HDC hdc, HWND hwndChild, int type)
 * GetItemListColor:  Get given color id # for given owner-drawn list box.
 *    (Inventory has different colors than popup dialog lists)
 *    Doesn't return color itself so that caller can use id to call GetBrush.
+*    Now colors magic items; any future item colors should be added here.
```

Comment:
"Now" will mean nothing to a reader in 10 years.  Better to remove that and instead document what "flags" does.